tests/src/scoreHandler.test.ts

```diff
@@ -0,0 +1,29 @@
+import { _testing } from "../../components/scoreHandler"
+import { ContractSession, RatingKill } from "../../components/types/types"
+
+describe("calculatePlaystyle", () => {
+    test("default", () => {
+        const contractSession = <ContractSession>{
+            kills: new Set<RatingKill>(),
+        }
+
+        const result = _testing.calculatePlaystyle(contractSession)
+
+        expect(result[0].Type).toBe("HEAD_SHOT_ASSASSIN")
+    })
+
+    test("pistol", () => {
+        const ratingKill: RatingKill = <RatingKill>{
+            KillClass: "ballistic",
+            KillItemCategory: "pistol",
+        }
+
+        const contractSession = <ContractSession>{
+            kills: new Set<RatingKill>([ratingKill]),
+        }
+
+        const result = _testing.calculatePlaystyle(contractSession)
```

Comment:
I'd be fine with exporting things like calculatePlaystyle - makes sense.  If we don't want people to use them (outside of tests), we can use `@internal`.